src/main/java/jetbrains/buildServer/iaa/AutoAssignerBuildFeature.java

```diff
@@ -51,6 +52,25 @@ public String getEditParametersUrl() {
     return myEditUrl;
   }
 
+  @NotNull
+  @Override
+  public String describeParameters(@NotNull Map<String, String> params) {
```

Comment:
Here also useful to see ins which mode auto-assigned was enabled: silent or normal